### ISSUE TITLE
If anyone wants to set back old github styles

### ISSUE THREAD

**Issue by bl00mber** [isaacs/github#1822]
https://github.com/bl00mber/github-old-styles

**Comment by probonopd** [comment]
Please bring back the original style before there were black header bars and popping green and blue colors.

**Comment by bl00mber** [comment]
I've added back buttons gradients. Not sure what is a bars and popping colors :sweat_smile:

**Comment by Lemmingh** [comment]
> black header bars and popping green and blue colors

Sounds like something that Google Play did a few years ago. 😄

**Comment by probonopd** [comment]
> Please bring back the original style before there were black header bars and popping green and blue colors.

2014 GutHub design, when at the top there was a grey (rather than black) bar, and when colors were subdued.

![github-primer](https://user-images.githubusercontent.com/2480569/85918055-f3796300-b84e-11ea-9bbe-e92980910faf.png)

**Comment by probonopd** [comment]
Good:

![good](https://user-images.githubusercontent.com/2480569/85918172-d85b2300-b84f-11ea-9b3d-ce1e38f143a3.png)

Bad:

![worst](https://user-images.githubusercontent.com/2480569/85918176-dc874080-b84f-11ea-8dff-0601fbead1a9.png)

(This is what I see if I put two browser windows alongside each other on a Full HD screen, a setup that I think is quite typical on desktop machines.)

**Comment by bl00mber** [comment]
Yeah it looks strange and I wonder why anyone will ever want to use GitHub on mobile phones? This is most likely can be reversed too, but I do not use two browser windows alongside so it's not my priority now (and I like a bit more space between UI elements). Feel free to send a PR though. The only reason why I have done a redesign is because new design is ridiculous (on my opinion), it transforms GitHub into... Skype clone? Seems someone at Microsoft inc. likes circles _too much_. This obsession over Flat design (which was developed at Microsoft)  elements is something vary Bad. The Google's Material UI or even Apple's iOS 3D design which was used on GitHub previously is better than this. On my opinion the Flat design is a total failure because it is tablet-oriented. Who develops software on tablets?

**Comment by bl00mber** [comment]
Disabled tablet navigation.
